src/tiled/mainwindow.cpp

```diff
@@ -1773,3 +1777,25 @@ void MainWindow::reloadError(const QString &error)
 {
     QMessageBox::critical(this, tr("Error Reloading Map"), error);
 }
+
+void MainWindow::setUseDarkTheme(bool useDarkTheme)
+{
+    if (useDarkTheme)
+    {
+        QFile darkThemeFile(QLatin1String(":darkstyle/style.qss"));
+        if (darkThemeFile.exists())
+        {
+            darkThemeFile.open(QFile::ReadOnly | QFile::Text);
+            setStyleSheet(QTextStream(&darkThemeFile).readAll());
+        }
+        else
+        {
+            QMessageBox::warning(this, tr("Error Loading Theme"), QLatin1String("Could not load the dark theme for Tiled"));
+        }
+    }
+    else
+    {
+        // Clears the stylesheet. I wonder if this will create issues with the style code in main.ccp?
+        setStyleSheet(QLatin1String(""));
+    }
+}
```

Comment:
Please do make sure to still end the file with a newline, to avoid the need for these markers.